### PR TITLE
Fix cs

### PR DESCRIPTION
**Pull request by wbloszyk**

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ /**
+ * DO NOT EDIT THIS FILE!
+ *
+ * It's auto-generated by sonata-project/dev-kit package.
+ */
+
+ $header = <<<'HEADER'
+This file is part of the <name> project.
+
+(c) <yourname> <youremail>
```
Comment (core23) [inline .php_cs.dist]: Should we use the sonata header here @sonata-project/contributors ?

Comment (OskarStark) [reply .php_cs.dist]: yes please

File: .php_cs [hunk 2]
```diff
@@ -1,40 +1,5 @@
 <?php
 
-/*
- * DO NOT EDIT THIS FILE!
- *
- * It's auto-generated by sonata-project/dev-kit package.
- *
- * Package `sllh/php-cs-fixer-styleci-bridge` is required to get it working.
- */
+// TO BE REMOVED.
```
Comment (core23) [inline .php_cs]: You can remove this file with this commit

File: src/Sonata/Bundle/DemoBundle/DataFixtures/ORM/LoadOrderData.php [hunk 3]
```diff
@@ -124,20 +126,32 @@ public function load(ObjectManager $manager)
         $manager->flush();
     }
 
+    /**
+     * {@inheritdoc}
```
Comment (core23) [inline src/Sonata/Bundle/DemoBundle/DataFixtures/ORM/LoadOrderData.php]: Please remove this for all files

Comment (wbloszyk) [reply src/Sonata/Bundle/DemoBundle/DataFixtures/ORM/LoadOrderData.php]: Done

**Comment by core23** [comment]
Thanks @wbloszyk

**Pull request merged**